src/main/resources/field_value_functions_axioms.smt2

```diff
@@ -11,24 +11,22 @@
 ; The axiom therefore needs to be emitted after the sort wrappers have
 ; been emitted.
 
-(assert (forall ((vs $FVF<$T$>) (ws $FVF<$T$>)) (!
+(assert (forall ((vs $FVF<$FLD$>) (ws $FVF<$FLD$>)) (!
```

Comment:
is it guaranteed that any name accepted for a field in Silver can just be replaced in this string such that it still produces valid smtlib2 code?